models/Schematic_DataModel.php

```diff
@@ -40,14 +40,21 @@ protected function defineAttributes()
      * Populate data model from yaml.
      *
      * @param string $yaml
+     * @param string $overrideYaml
      *
      * @return Schematic_DataModel
      */
-    public static function fromYaml($yaml)
+    public static function fromYaml($yaml, $overrideYaml)
     {
         $data = Yaml::parse($yaml);
+        $overrideData = Yaml::parse($overrideYaml);
+        if ($overrideData != null) {
+            $mergedData = array_replace_recursive($data, $overrideData);
```

Comment:
I think its better to use `\CMap::mergeArray` here instead of array_replace_recursive, because that will really "override" instead of "ignore"